compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -183,6 +183,60 @@ struct ConvertTensorFromElementsPattern
   }
 };
 
+/// Returns a sizes array with the dynamic dims.
+static SmallVector<Value> getDynamicTensorSizes(OpBuilder &builder,
+                                                Location loc, ShapedType stp,
+                                                Value tensor) {
+  SmallVector<Value> sizes;
+  for (const auto &d : enumerate(stp.getShape())) {
```

Comment:
let's use structured binding, it is more common in IREE/MLIR codebase.

```suggestion
  for (const auto [idx, size] : enumerate(stp.getShape())) {
```